webapp/src/actions.js

```diff
@@ -53,29 +53,6 @@ export function updateRhsState(rhsState) {
     };
 }
 
-// TODO: Move this into mattermost-redux or mattermost-webapp.
-export const getPluginServerRoute = (state) => {
-    const config = getConfig(state);
-
-    let basePath = '';
-    if (config && config.SiteURL) {
-        basePath = new URL(config.SiteURL).pathname;
-
-        if (basePath && basePath[basePath.length - 1] === '/') {
-            basePath = basePath.substr(0, basePath.length - 1);
-        }
-    }
-
-    return basePath + '/plugins/' + pluginId;
-};
-
-export const telemetry = (event, properties) => async (dispatch, getState) => {
-    await fetch(getPluginServerRoute(getState()) + '/telemetry', Client4.getOptions({
-        method: 'post',
-        body: JSON.stringify({event, properties}),
-    }));
-};
-
```

Comment:
Is it worth removing the unused method? We might want to add telemetry at some point.